waspc/data/Generator/templates/server/src/auth/providers/index.ts

```diff
@@ -1,18 +1,15 @@
 {{={= =}=}}
-
-import { join } from 'path'
 import { Router } from "express";
 
-import { getDirPathFromFileUrl, importJsFilesFromDir } from "wasp/server/utils";
-
-import { ProviderConfig } from "wasp/auth/providers/types";
+{=# providers =}
+{=& importStatement =}
+{=/ providers =}
 
-const whitelistedProviderConfigFileNames = [
-  {=# enabledProviderIds =}
-  "{= . =}.js",
-  {=/ enabledProviderIds =}
+const providers = [
+  {=# providers =}
+  {= importIdentifier =},
+  {=/ providers =}
 ];
-const providers = await importProviders(whitelistedProviderConfigFileNames);
```

Comment:
Same reason for removing as https://github.com/wasp-lang/wasp/pull/1714/files#r1474701889